src/main/java/frc/robot/elevator/ElevatorSubsystem.java

```diff
@@ -49,6 +51,16 @@ private static double clampHeight(double height) {
   private double averageMotorCurrent;
   private final CoastOut coastRequest = new CoastOut();
 
+  private final TrapezoidProfile motionProfile =
+      new TrapezoidProfile(
+          new TrapezoidProfile.Constraints(
+              RobotConfig.get().elevator().leftMotorConfig().MotionMagic.MotionMagicCruiseVelocity,
+              RobotConfig.get().elevator().leftMotorConfig().MotionMagic.MotionMagicAcceleration));
+
+  TrapezoidProfile.State goalSetPoint = new TrapezoidProfile.State();
+  TrapezoidProfile.State currentSetPoint = new TrapezoidProfile.State();
+  double syncingCoeffecient = 1.0;
```

Comment:
this is not how we want to implement sync, but that's fine for now. the actual way will be by constructing a new trapezoidal motion profile with tighter constraints, to artificially slow down the elevator so it follows the arm.

please don't write this code though. probably delete all the sync stuff and just leave a TODO that says where to implement the sync